### PR TITLE
JBPM-6729: Generate correct base-path when context-root is empty (for example wh…

### PR DESCRIPTION
**Pull request by DuncanDoyle**
…en we deploy KIE-Server as ROOT.war, which we do in our OpenShift images).

Fixes JBPM-6729

### CODE REVIEW

File: kie-server-parent/kie-server-services/kie-server-services-swagger/src/test/java/org/kie/server/services/swagger/SwaggerKierServerExtensionTest.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
+package org.kie.server.services.swagger;
```
Comment (mswiderski) [inline kie-server-parent/kie-server-services/kie-server-services-swagger/src/test/java/org/kie/server/services/swagger/SwaggerKierServerExtensionTest.java]: missing license header

**Comment by mswiderski** [comment]
@DuncanDoyle and commit message with JIRA number please :)

**Comment by DuncanDoyle** [comment]
Should be good now.

**Comment by mswiderski** [comment]
@DuncanDoyle license header should be first thing in the file not after package... I think so :)

**Comment by DuncanDoyle** [comment]
Djeeezzz, that happens when you do 20 things at the same time ...... Fixed now.

**Pull request merged**